docs/kyma/10-04-common-errors.md

```diff
@@ -69,3 +69,7 @@ To resolve this error, upgrade Tiller. Run:
 ```
 kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/{YOUR_KYMA_VERSION}/installation/resources/tiller.yaml
 ```
+
+## Maximum number of retries reached
+
+Kyma Installer retries installing a failed release a given number of times (default is 5). It stops the installation when it reaches the limit and returns this message: `Max number of retries reached during step {STEP_NAME}`. To start it again, check Kyma Installer logs to find out why this step failed, fix the issue, and follow the [update process](#installation-update-kyma).
```

Comment:
Although it's already merged, I will leave a comment here @colunira : Instead of linking to update procedure, which explains things like updating docker image I would simply just point `kubectl -n default label installation/kyma-installation action=install` as trigger command for starting installation process again :)